data/models/4x-SRFormer_SRx4_DF2K.json

```diff
@@ -0,0 +1,34 @@
+    "date": "2022-11-24",
+    "architecture": "SRFormer",
+    "size": [
+        "base",
+        "img_size: 48",
+        "window_size: 22"
+    ],
```

Comment:
@joeyballentine What do we do about this? This doesn't exactly look like the sizes of an ESRGAN model. Is this correct, is it wrong?

Man, we really need to automate detecting this.

Reply:
That's because it's not an esrgan model. It has a fundamentally different way of defining it's size. It's more similar to SwinIR which you can see here: https://github.com/OpenModelDB/open-model-database/blob/a98c9b00f19476e893977ee1949de9a80198413c/data/models/3x-classicalSR-DF2K-s64w8-SwinIR-M.json#L16

There's really no standard for this, so to me I'm fine leaving it in whatever format is closest to the official size definitions from the repo

Reply:
Note: s and w from SwinIR are the img_size and window from here

Reply:
ah yea I had manually added these parameters. The base one isnt really needed, I could actually take it out again, thats just for me to distinguish, I call it a base model for example if its SRFormer but not lightweight, or if its HAT but not HAT-S or HAT-L. Why I have it in there specifically is because there currently is kind of a conflict I recognized with SRFormer. So in their official code repo they have an official training config. Their network parameters in there differ from the ones they used for their official paper pretrains, or in other words their official test config file has different network parameters than their train config file. The only thing they differ in are these two parameters, this is why I added them manually there, because there is currently no way of knowing, neither for me nor anybody else, which values someone will have chosen when having trained an SRFormer base model. This inconsitency does not happen with SRFormer_light. Hope this helps to understand why I added these parameters manually there.